MusicNotationKit/MusicNotationKit/Tuplet.swift

```diff
@@ -18,13 +18,27 @@ public struct Tuplet: NoteCollection {
         }
     }
     /// The number of notes of the specified duration that this tuplet contains
-    public var noteCount: Int { return notes.count }
+    public let noteCount: Int
     /// The duration of the notes that define this tuplet
     public let noteDuration: NoteDuration
     /// The number of notes that this tuplet fits in the space of
     public let noteTimingCount: Int
+    public var first: Note {
+        do {
+            return try note(at: 0)
+        } catch {
+            fatalError(String(describing: error))
```

Comment:
Yeah. I didn't want to have this be nil, since it should totally never happen. I feel like this one is ok to crash since it is so unlikely.